_dev/src/components/product-feed/settings/attribute-mapping/attribute-field.vue

```diff
@@ -100,7 +102,9 @@ export default {
         this.field.mapped = value;
       },
     },
-
+    attributeToEdit() {
+      return this.$route.hash.substring(1);
```

Comment:
I guess `hash` can be null?

Reply:
yes but I don't see any error in my console. Do you think of a case it can crash ?